src/csv_reporter.cc

```diff
@@ -22,27 +22,31 @@
 #include <tuple>
```

Comment:
split the CSV changes out to another PR. I don't know that we need it, and it's helpful to keep this PR more focused.